qutebrowser/browser/adblock.py

```diff
@@ -111,9 +111,9 @@ def __init__(self):
 
         config.instance.changed.connect(self._update_files)
 
-    def is_blocked(self, url):
+    def is_blocked(self, url, primary=None):
```

Comment:
Not sure `primary` is a good name for the argument - why not `first_party_url` to be consistent with what QtWebEngine uses?